.github/workflows/build.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           $CONDA/bin/pip install -e .
       - name: linting [flake8]
         run: |
-          $CONDA/bin/flake8 --ignore=D,W503,W504 ioSPI tests
+          $CONDA/bin/flake8 --ignore=D,W503,W504,BLK100 ioSPI tests
```

Comment:
Nice! What does BLK100 do?

Reply:
"Black would make changes." I wonder if it's too heavy handed...

I made some style issues locally and checked how black and flake8 react. 

When I run flake8 locally with the `--ignore=BLK100` flag, I still get errors from flake8.

So I think it's a useful flag when there are discrepancies between flake8 and black.

Reply:
I looked into things a bit more. 

I removed this flag and looked up the areas that cause a failure in the automated build.

These areas cause a problem in black with the default line length (in black it's 80 something). However flake8 enforces 79 characters. Locally, I use `black -l 79` to transform code so that it passes flake8. But then on github, the flake8-black complains. So the BLK100 is turning this off. However, we really want to pass in the 79 default to the black within flake8-black, rather than turning off all of black.


Here we see that blacking things with the longer line default makes flake8 compalin
```
(compspi_py3) gws-MacBook-Pro-6:ioSPI gw$ black transfer.py --diff 
--- transfer.py	2021-08-24 20:02:38.060064 +0000
+++ transfer.py	2021-08-24 20:29:59.742021 +0000
@@ -40,13 +40,11 @@
     freq_mag_2d = rho * psize
     return (freq_mag_2d, angles_rad)


 @nb.jit(cache=True, nopython=True, nogil=True)
-def eval_ctf(
-    s, a, def1, def2, angast=0, phase=0, kv=300, ac=0.1, cs=2.0, bf=0, lp=0
-):
+def eval_ctf(s, a, def1, def2, angast=0, phase=0, kv=300, ac=0.1, cs=2.0, bf=0, lp=0):

(compspi_py3) gws-MacBook-Pro-6:ioSPI gw$ black  transfer.py
reformatted transfer.py
All done! ✨ 🍰 ✨
1 file reformatted.
(compspi_py3) gws-MacBook-Pro-6:ioSPI gw$ flake8 transfer.py
transfer.py:45:80: E501 line too long (86 > 79 characters)

```

If we black with the 79 character default of flake8, then things are fine locally. However, flake8-black gives that `BLK100 error`, just like it would complain with out the `-l 79` flag.
```
(compspi_py3) gws-MacBook-Pro-6:ioSPI gw$ black transfer.py --diff -l 79 
All done! ✨ 🍰 ✨
1 file would be left unchanged.
(compspi_py3) gws-MacBook-Pro-6:ioSPI gw$ flake8 transfer.py # no changes requested
```

Reply:
So for now I'll keep the BLK100 flag in.